src/main.jsx

```diff
@@ -11,7 +10,6 @@ ReactDOM.createRoot(document.getElementById('root')).render(
   <React.StrictMode>
     <Provider store={store}>
       <App />
-      <ToastContainer/>
     </Provider>
```

Comment:
why did you remove ToastContainer???

Reply:
it is already in the login and signup pages adding it here displayed it twice